README.md

```diff
@@ -10,7 +10,7 @@ some new part of the compiler that they haven't worked on before.
 
 You may also find the rustdocs [for the compiler itself][rustdocs] useful.
 
-[rustdocs]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/
+[rustdocs]: https://doc.rust-lang.org/nightly/nightly-rustc
```

Comment:
Interesting, I didn't know such a page existed

Reply:
I actually suggested this change a while ago (I think on Zulip?) but now that I think about it, it might be worse for new contributors to link to the index. `rustc_middle` has some helpful docs about HIR and MIR, while the index is just a giant list of crates. What do you think?

Reply:
I think we should suggest using the search button and note that the rustdoc's aren't meant to be an intro.

Reply:
Added that to the README :)